website/project/views/node.py

```diff
@@ -1008,6 +1008,66 @@ def get_children(auth, node, **kwargs):
     return _render_nodes(nodes, auth)
 
 
+def node_child_tree(user, node_ids):
```

Comment:
Recursing through a node tree is a very common operation. Can you reuse code from other parts of the OSF? I know, for one, the wiki table of contents has to do this. There are other places as well.